src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -23,6 +23,10 @@
  */
 abstract class BaseNode implements NodeInterface
 {
+    const DEFAULT_PATH_SEPARATOR = '.';
+
+    private $pathSeparator;
```

Comment:
I would keep it private if it does not need to be protected. It is easier for our BC policy